src/components/CardGrid/CardGrid.tsx

```diff
@@ -55,7 +55,7 @@ function CardGrid({
   onCardHover,
 }: CardGridProps) {
   const breakpoint: Breakpoint = useBreakpoint();
-  const posterAspect = parseAspectRatio(playlist.shelfImageAspectRatio);
+  const posterAspect = parseAspectRatio(playlist.aspectRatio || playlist.shelfImageAspectRatio);
```

Comment:
@ChristiaanScheermeijer Do you know any cases where `shelfImageAspectRatio` is used by customers? I would just leave `aspectRatio`.

Reply:
I don't mind keeping the more verbose param. What was the reason for the name change in the first place? Just `aspectRatio` might imply the sizing of the shelf itself.

Reply:
We use now `cardImage` instead of `shelfImage` as we changed the label to `card` by default (AC). Previously `shelfImage` custom param was used to define the label. I can also use `cardImageAspectRatio`.

Reply:
I don't think so. We're using this in the demo app configs, though. But if you are going to change it and remove the fallback, it would be good to mention it in the changelogs.

Reply:
Left the fallback with`shelfImageAspectRatio` (deprecated), added  `cardImageAspectRatio`